### PR TITLE
Add 2024 IWD Materials

### PR DESCRIPTION
**Pull request by nrennie**
- Add text for social media posts
- Add images for posts
- Add R code used to make images

### CODE REVIEW

File: 2024-social-media-posts.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+💜 We already have two new chapters with their first events this year!
+#IWD2024 #RLadies
+
+## Post 4
+Image: R-Ladies logo (or IWD gif)
+Text:
+How can you help R-Ladies as an ally? 
+💜 Support your local chapter with a free venue and sponsorship
+💜 Invitations to speaks at conferences and highlighting job opportunities
+💜 Volunteer as a conference abstract reviewer: https://guide.rladies.org/comm/abstract-review/
```
Comment (shannonpileggi) [inline 2024-social-media-posts.md:41]: flagging this one to @averi-giudicessi as we had discussed this one previoulsy.

File: 2024-social-media-posts.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Post 4
+Image: R-Ladies logo (or IWD gif)
+Text:
+How can you help R-Ladies as an ally? 
+💜 Support your local chapter with a free venue and sponsorship
+💜 Invitations to speaks at conferences and highlighting job opportunities
+💜 Volunteer as a conference abstract reviewer: https://guide.rladies.org/comm/abstract-review/ 
+💜 Consult our R-Ladies directory: https://rladies.org/directory/ 
+💜 Donate to the global R-Ladies organization
```
Comment (shannonpileggi) [inline 2024-social-media-posts.md]: We should add a link: https://www.paypal.com/donate/?hosted_button_id=VB4EK67MJ373E

@drmowinckels should we create an rladies redirect to the paypal?

**Comment by shannonpileggi** [comment]
thank you so much for preparing this @nrennie, this looks great so far!!

**Comment by d4tagirl** [comment]
This looks great! Should I wait for the PR to be merged so I can start preparing the posts?

**Comment by yabellini** [comment]
We have been highlighting as a community and two people from R-Ladies for another community because of IWD, I will share here the links to the blog posts so we can also post and share them.

**Pull request merged**